core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -74,10 +74,10 @@ public static function getTemplate($strTemplate)
 		// Check for a theme folder
 		if (\defined('TL_MODE') && TL_MODE == 'FE')
 		{
-			/** @var PageModel $objPage */
+			/** @var PageModel|null $objPage */
 			global $objPage;
 
-			if ($objPage->templateGroup)
+			if (null !== $objPage && $objPage->templateGroup)
```

Comment:
```suggestion
			if ($objPage->templateGroup ?? null)
```

Reply:
Does in this case the null coalescing Operator not crash if objPage itself is null?

Reply:
No it won't crash: https://3v4l.org/hlVR1 :)

Reply:
Crazy but does what it's supposed to ... Never used the Coalescing Operator in this way :-) "Again what learned" (as Lothar would say ). Thanks